src/Compilers/CSharp/Portable/Binder/LocalScopeBinder.cs

```diff
@@ -338,6 +368,14 @@ protected override LocalFunctionSymbol LookupLocalFunction(SyntaxToken nameToken
             return base.LookupLocalFunction(nameToken);
         }
 
+        internal override uint LocalScopeDepth
```

Comment:
Nit: consider expression-bodied property (=>)

Reply:
It was not as simple initially, but then I moved the logic to the ctor.

---
In reply to: [135359975](https://github.com/dotnet/roslyn/pull/21334#discussion_r135359975) [](ancestors = 135359975)